packages/auth-browser/src/lib/chains/eth.ts

```diff
@@ -389,7 +389,7 @@ export const connectWeb3 = async ({
 
   const provider = await dialog.getWalletProvider();
 
-  log('got provider', provider);
+  log('got provider');
```

Comment:
why did we remove the `provider` var in the log?

Reply:
the provider is a complex object with multiple circular references within. logging it seems unnecessary so i removed it. Since it is of type `{unknown}` we cannot know what properties are on this object so i removed the logging of it all together.